src/styles/product.scss

```diff
@@ -0,0 +1,144 @@
+.product {
+  background-color: $page-backgroun-color;
+  overflow: scroll;
```

Comment:
Avoid using the overflow property with value scroll, as it can cause unwanted scrollbars to appear. Consider using overflow: auto instead.